src/cascadia/TerminalControl/SearchBoxControl.xaml

```diff
@@ -0,0 +1,122 @@
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    x:Name="Root"
+    HorizontalAlignment="Stretch"
+    VerticalAlignment="Top"
+    d:DesignHeight="75"
+    d:DesignWidth="200"
+    TabNavigation="Cycle">
+
+    <UserControl.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.ThemeDictionaries>
+                <ResourceDictionary x:Key="Default">
+                    <Style x:Key="FontIconStyle" TargetType="FontIcon">
+                        <Setter Property="FontSize" Value="15" />
+                        <Setter Property="Foreground" Value="White" />
+                    </Style>
+                    <Style x:Key="ToggleButtonStyle" TargetType="ToggleButton">
+                        <Setter Property="Width" Value="20" />
+                        <Setter Property="Height" Value="20" />
+                        <Setter Property="Background" Value="Transparent" />
+                    </Style>
+                    <Style x:Key="SearchBoxBackground" TargetType="StackPanel">
+                        <Setter Property="Background" Value="Black" />
+                    </Style>
+                </ResourceDictionary>
+                <ResourceDictionary x:Key="Light">
+                    <Style x:Key="FontIconStyle" TargetType="FontIcon">
+                        <Setter Property="FontSize" Value="15" />
+                        <Setter Property="Foreground" Value="White" />
+                    </Style>
+                    <Style x:Key="ToggleButtonStyle" TargetType="ToggleButton">
+                        <Setter Property="Width" Value="20" />
+                        <Setter Property="Height" Value="20" />
+                        <Setter Property="Background" Value="Transparent" />
+                    </Style>
+                    <Style x:Key="SearchBoxBackground" TargetType="StackPanel">
+                        <Setter Property="Background" Value="Black" />
+                    </Style>
+                </ResourceDictionary>
+                <ResourceDictionary x:Key="HighContrast">
+                    <Style x:Key="FontIconStyle" TargetType="FontIcon">
+                        <Setter Property="FontSize" Value="15" />
+                        <Setter Property="Foreground" Value="{ThemeResource SystemColorWindowTextColor}" />
+                    </Style>
+                    <Style x:Key="ToggleButtonStyle" TargetType="ToggleButton">
+                        <Setter Property="Width" Value="20" />
+                        <Setter Property="Height" Value="20" />
+                        <Setter Property="Background" Value="Transparent" />
+                    </Style>
+                    <Style x:Key="SearchBoxBackground" TargetType="StackPanel">
+                        <Setter Property="Background" Value="{ThemeResource SystemColorWindowColor}" />
+                    </Style>
+                </ResourceDictionary>

```

Comment:
I don't know xaml too well but a lot of this code is repeated. Is there any way to cut down on that?

Reply:
moved ToggleButtonStyle and ButtonStyle as shared